recipe/conda_build_config.yaml

```diff
@@ -1,2 +0,0 @@
-channel_sources:
-  - conda-forge,defaults
```

Comment:
I think we unfortunately need to have both channel sources, at least for now, as on Windows there isn't a remotely-decent tensorflow available in conda-forge. We're working on migrating off tensorflow, but we're not done yet.

Reply:
Yeah, I noticed. The problem is that conda-forge isn't geared to be using any other channel_sources than itself (or at least: doesn't want to be anymore). Getting tensorflow and pytorch built on windows are two long-standing items we want to tackle; pytorch is actually getting close, and hopefully that'll translate to tensorflow soon after.

In any case, depending on the outcome of https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/339, you'll probably have to remove windows builds here until we get tensorflow sorted out (and any help for that is welcome!)

Reply:
Presumably removing it from builds that don't need it (meaning other packages) could be done now, with removing it from the allowable configs something that could hold off until it doesn't break anything? We have a lot of users on Windows and losing the ability to package for that platform would be bad for us as a project.